### PR TITLE
(feat) O3-3246 - Ward App - open patient record in workspace

### PR DESCRIPTION
**Pull request by brandones**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary

The skeleton of a patient workspace in the ward app.

## Screenshots

I don't know what's going on with the patient info in the cards but it's unrelated to this ticket.

https://github.com/openmrs/openmrs-esm-patient-management/assets/1031876/def7c9df-df6d-412a-b1f7-d0b826ef177c

## Related Issue

https://openmrs.atlassian.net/browse/O3-3246

### CODE REVIEW

File: packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx [hunk 1]
```diff
@@ -13,6 +15,14 @@ const WardPatientCard: React.FC<WardPatientCardProps> = (props) => {
       {patientCardRows.map((WardPatientCardRow, i) => (
         <WardPatientCardRow key={i} {...props} />
       ))}
+      <button
+        className={styles.wardPatientCardButton}
+        onClick={() =>
+          launchWorkspace<WardPatientWorkspaceProps>('ward-patient-workspace', { patientUuid: props.patient.uuid })
+        }>
+        {/* Name will not be displayed; just there for a11y */}
```
Comment (mseaton) [inline packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx:23]: Not sure what this means, but ok

File: packages/esm-ward-app/src/ward-patient-workspace/patient-history-card.component.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (mseaton) [inline packages/esm-ward-app/src/ward-patient-workspace/patient-history-card.component.tsx:1]: What is this component for?  Is it related to this ticket?

File: packages/esm-ward-app/src/ward-patient-workspace/ward-patient-action-button.extension.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (mseaton) [inline packages/esm-ward-app/src/ward-patient-workspace/ward-patient-action-button.extension.tsx:1]: It is unclear if or where this is used here.  Is this related to this feature?

File: packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+export interface WardPatientWorkspaceProps extends DefaultWorkspaceProps {
+  patientUuid: string;
+}
+
+export default function WardPatientWorkspace({ patientUuid, setTitle }: WardPatientWorkspaceProps) {
+  const { patient, isLoading, error } = usePatient(patientUuid);
+
+  useEffect(() => {
+    if (isLoading) {
+      setTitle('Ward Patient', <InlineLoading />);
```
Comment (mseaton) [inline packages/esm-ward-app/src/ward-patient-workspace/ward-patient.workspace.tsx:25]: This should probably be translated, no?  And for the error case below too.

**Review by mseaton — APPROVED** [review]
Basically LGTM, a few minor comments / questions.

File: packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx (hunk 1 above)
Comment (brandones) [reply packages/esm-ward-app/src/ward-patient-card/ward-patient-card.tsx:23]: a11y = accessibility. The name serves as a label for the box/button so that the browser knows what it is.

File: packages/esm-ward-app/src/ward-patient-workspace/patient-history-card.component.tsx (hunk 2 above)
Comment (brandones) [reply packages/esm-ward-app/src/ward-patient-workspace/patient-history-card.component.tsx:1]: Oh yeah, stubbed that out before I realized it wasn't asked for in the ticket. Removing it.

File: packages/esm-ward-app/src/ward-patient-workspace/ward-patient-action-button.extension.tsx (hunk 3 above)
Comment (brandones) [reply packages/esm-ward-app/src/ward-patient-workspace/ward-patient-action-button.extension.tsx:1]: Things with the `.extension.tsx` suffix are extensions, which are used in the [index file](https://github.com/openmrs/openmrs-esm-patient-management/blob/e82d2c90aa9a28140659bbec69da0afcd130328b/packages/esm-ward-app/src/index.ts#L29). If something has the extension suffix and isn't used in the index file, then something has gone wrong.
